compute_endpoint/globus_compute_endpoint/endpoint/interchange.py

```diff
@@ -338,8 +350,8 @@ def process_pending_tasks() -> None:
                     d_tag, prop_headers, body = self.pending_task_queue.get(timeout=1)
                     task_q_subscriber.ack(d_tag)
 
-                    fid: str = prop_headers.get("function_uuid")
-                    tid: str = prop_headers.get("task_uuid")
+                    fid: str | None = prop_headers.get("function_uuid")
+                    tid: str | None = prop_headers.get("task_uuid")
 
                     if not fid or not tid:
                         raise InvalidMessageError(
```

Comment:
Did `mypy` require this change?  Because the if-check just below (currently line -344/+355) will require it regardless, and is (currently) an expectation of the packet shape via the incoming task queue.  Seems to me that these should be (type) required.

Reply:
Line 356 implies that it could be `None`, but I see your point. Just pushed an update.